app/generate_csv_eventyay.py

```diff
@@ -0,0 +1,49 @@
+import urllib.request, json, os
+
+APP_ROOT = os.path.dirname(os.path.abspath(__file__))
+UPLOAD_FOLDER = os.path.join(APP_ROOT, 'static/uploads')
+MAX_STRING = "chaftsladen Potsdam (F"
+
+def tocsv(url_eventyay):
+    with urllib.request.urlopen(url_eventyay) as url:
+        data = json.loads(url.read().decode(url.info().get_param('charset') or 'utf8'))
```

Comment:
Just as a note, this line is perfect. I also think, JSON is always UTF-8 encoded.